java/pending_code_changes.txt

```diff
@@ -1 +1,5 @@
-
+Code changes:
+ - Added to checked-in tools/java/cpp-build and tools/java/java-build jars
```

Comment:
I am surprised that these depend on PhoneNumberUtil at all; it's a bit of a circular dependency, is it not? However I believe you - I remember having problems with this in the past. We should look at removing it...

Question though: do we want to update pending_code_changes.txt when we update these jars? they aren't really part of the project, they are just a tool to build the metadata. I don't think anyone is using these (or any other) methods through these jars: and indeed they shouldn't be..!